filter/options.go

```diff
@@ -13,4 +13,5 @@ type Options struct {
 	Prompt         string       `help:"Prompt to display" default:"> "`
 	PromptStyle    style.Styles `embed:"" prefix:"prompt." set:"defaultForeground=240" set:"name=prompt"`
 	Width          int          `help:"Input width" default:"20"`
+    Delimeter      string       "help:\"Delimeter to split input on\" default:\"\n\""
```

Comment:
Did you mean
```suggestion
    Delimeter      string       `help:"Delimeter to split input on" default:"\n"`
```

Reply:
If you do that. Than it sets the default delimiter to a literal "\n" instead of a newline

Reply:
Are `"` delimited field tags legal in Go?
Would the following work?
```suggestion
    Delimeter      string       `help:"Delimeter to split input on" default:"
"`
```

Reply:
I don't know, but the code I submitted compiled and ran like I expected. Let me try what you put.

Reply:
alright, what you posted worked, I didn't want to do that though because now there is a newline that can't be removed without breaking the code - at least the first is explicit, even if it is messy.